main.py

```diff
@@ -44,3 +46,21 @@ def create():
         return list_all()
     except ValidationError as err:
         return jsonify(err.messages), 400
+
+
+@app.route('/v1/todos/<identifier>', methods=['DELETE'])
+def delete(identifier):
+    try:
+        todos_repo.remove(int(identifier))
+        return jsonify(identifier)
+    except IndexError as err:
+        return jsonify(str(err)), 400
```

Comment:
When you navigate to a blog page via a link and that page is missing (meanwhile removed), what will you get?